packages/compiler/src/collector/collect.ts

```diff
@@ -10,15 +8,20 @@ import { match } from "ts-pattern";
 import { decode } from "./decode";
 import { handleJsxExpression } from "./expression";
 import { extractPseudoAttribute } from "./pseudo";
+import {
+  UnevaluatedConditionalExpression,
+  unevaluatedConditionalExpression,
+} from "../static-branching";
 
 export const collectPropsFromJsx = (
   node: JsxOpeningElement | JsxSelfClosingElement
-) => {
+): Record<string, any> => {
   const jsxAttributes = node.getAttributes();
   const extracted: Record<string, any> = {};
+
   jsxAttributes.forEach((jsxAttribute) => {
     if (Node.isJsxAttribute(jsxAttribute)) {
-      const propName = jsxAttribute.getNameNode().getFullText();
+      const propName = jsxAttribute.getNameNode().getText();
```

Comment:
Note: @poteboy I guess the usage of `getFullText` was probably a mistake. Due to this we have several `trim()` in compiler package. I did not include the removals of `trim()` in this PR but we can remove them as well in later PRs

Reply:
Ah I see, thaks this helps a lot :)